applications/plugins/SofaDistanceGrid/src/SofaDistanceGrid/components/collision/DistanceGridCollisionModel.cpp

```diff
@@ -474,31 +475,31 @@ void FFDDistanceGridCollisionModel::init()
     ffdSGrid = dynamic_cast< topology::SparseGridTopology* > (ffdMesh);
     if (!ffd || (!ffdRGrid && !ffdSGrid))
     {
-        serr <<"FFDDistanceGridCollisionModel requires a Vec3-based deformable model with associated RegularGridTopology or SparseGridTopology" << sendl;
+        msg_error() << "FFDDistanceGridCollisionModel requires a Vec3-based deformable model with associated RegularGridTopology or SparseGridTopology";
         return;
     }
 
     DistanceGrid* grid = NULL;
     if (fileFFDDistanceGrid.getValue().empty())
     {
-        serr<<"ERROR: FFDDistanceGridCollisionModel requires an input filename" << sendl;
+        msg_error() << "ERROR: FFDDistanceGridCollisionModel requires an input filename";
         return;
     }
-    sout << "FFDDistanceGridCollisionModel: creating "<<nx.getValue()<<"x"<<ny.getValue()<<"x"<<nz.getValue()<<" DistanceGrid from file "<<fileFFDDistanceGrid.getValue();
-    if (scale.getValue()!=1.0) sout<<" scale="<<scale.getValue();
-    if (sampling.getValue()!=0.0) sout<<" sampling="<<sampling.getValue();
-    if (box.getValue()[0][0]<box.getValue()[1][0]) sout<<" bbox=<"<<box.getValue()[0]<<">-<"<<box.getValue()[0]<<">";
-    sout << sendl;
+    msg_info() << "FFDDistanceGridCollisionModel: creating "<<nx.getValue()<<"x"<<ny.getValue()<<"x"<<nz.getValue()<<" DistanceGrid from file "<<fileFFDDistanceGrid.getValue();
```

Comment:
It is probably good to remove all the FFDDIstanceGridCollisionModel... as the type of object is already contained in msg_info(this)